deployments/pulse3d/services/pulse3d/src/core/config.py

```diff
@@ -3,7 +3,7 @@
 
 config = Config(".env")
 # version used to tag the docker image
-VERSION = "0.6.1"
+VERSION = "0.6.2"
```

Comment:
Does the makefile match this version?

Reply:
Nikita actually just bumped these anyway, so I'll increase everything